dr-multi-replica.md

```diff
@@ -0,0 +1,212 @@
+
+1. Create a topology file similar to the following:
+
+    ```toml
+    global:
+      user: "root"
+      ssh_port: 22
+      deploy_dir: "/data/tidb_cluster/tidb-deploy"
+      data_dir: "/data/tidb_cluster/tidb-data"
+
+    server_configs:
+      tikv:
+        server.grpc-compression-type: gzip
+      pd:
+        replication.location-labels:  ["Region","AZ"] # PD schedules replicas according to the Region and AZ configuration of TiKV nodes.
+
+    pd_servers:
+      - host: tidb-dr-test1
+        name: "pd-1"
+      - host: tidb-dr-test2
+        name: "pd-2"
+      - host: tidb-dr-test3
+        name: "pd-3"
+      - host: tidb-dr-test4
+        name: "pd-4"
+      - host: tidb-dr-test5
+        name: "pd-5"
+
+    tidb_servers:
+      - host: tidb-dr-test1
+      - host: tidb-dr-test3
+
+
+    tikv_servers:  # Label the Regions and AZs of each TiKV node through the labels option.
+      - host: tidb-dr-test1
+        config:
+          server.labels: { Region: "Region1", AZ: "AZ1" }
+      - host: tidb-dr-test2
+        config:
+          server.labels: { Region: "Region1", AZ: "AZ2" }
+      - host: tidb-dr-test3
+        config:
+          server.labels: { Region: "Region2", AZ: "AZ3" }
+      - host: tidb-dr-test4
+        config:
+          server.labels: { Region: "Region2", AZ: "AZ4" }
+      - host: tidb-dr-test5
+        config:
+          server.labels: { Region: "Region3", AZ: "AZ5" }
+
+          raftstore.raft-min-election-timeout-ticks: 1000
+          raftstore.raft-max-election-timeout-ticks: 1200
+
+    monitoring_servers:
+      - host: tidb-dr-test2
+
+    grafana_servers:
+      - host: tidb-dr-test2
+
+    alertmanager_servers:
+      - host: tidb-dr-test2
+      ```
+
+    The preceding configurations use the following options to optimize toward cross-Region DR:
+
+    - `server.grpc-compression-type: gzip` to enable gRPC message compression in TiKV, thus reducing network traffic.
+    - `raftstore.raft-min-election-timeout-ticks` and `raftstore.raft-max-election-timeout-ticks` to extend the time before Region 3 participates in the election, thus preventing any replica in this Region from being voted as the leader.
+
+2. Create a cluster using the preceding configuration file:
+
+    ```toml
+    # tiup cluster deploy drtest v6.4.0 ./topo.yaml
+    # tiup cluster start drtest --init
+    # tiup cluster display drtest
+    ```
+
+    Configure the number of replicas and the leader limit for the cluster:
+
+    ```toml
+    # tiup ctl:v6.4.0 pd config set max-replicas 5
+    # tiup ctl:v6.4.0 pd config set label-property reject-leader Region Region3
+
+    # The following step adds some test data to the cluster, which is optional.
+    # tiup bench tpcc  prepare -H 127.0.0.1 -P 4000 -D tpcc --warehouses 1
+    ```
```

Comment:
```suggestion
    ```shell
    tiup ctl:v6.4.0 pd config set max-replicas 5
    tiup ctl:v6.4.0 pd config set label-property reject-leader Region Region3

    # The following step adds some test data to the cluster, which is optional.
    tiup bench tpcc  prepare -H 127.0.0.1 -P 4000 -D tpcc --warehouses 1
    ```
```